third_party/openmetadata/install_OM.sh

```diff
@@ -28,7 +28,7 @@ tmp_dir=$(mktemp -d)
 # download files to temp directory
 cd $tmp_dir
 
-files_to_download=(Makefile Makefile.env pv1.yaml pv2.yaml values-deps.yaml)
+files_to_download=(Makefile Makefile.env pv1.yaml pvc1.yaml pv2.yaml pvc2.yaml pvc3.yaml pvc4.yaml values-deps.yaml)
```

Comment:
These files are in the same git repo we are running make targets from. I don't understand the meaning of the curl command.

Reply:
The idea is to allow people to install `OpenMetadata` without cloning the `fybrik` repository, by running:
```
curl https://raw.githubusercontent.com/fybrik/fybrik/master/third_party/openmetadata/install_OM.sh | bash -
```

Reply:
But then you don't really allow making any changes to the makefile - the command will automatically download and run make. Provide an argument for specifying the release namespace, perhaps? Which will be optional, with a default value.

Reply:
You can set the `INSTALLATION_NAMESPACE` environment variable (e.g., to `omd`) and run
```
curl https://raw.githubusercontent.com/fybrik/fybrik/master/third_party/openmetadata/install_OM.sh | bash -
```
and the script will install OpenMetadata to the namespace of your choice